JSON_for_IO-Link.yaml

```diff
@@ -1125,16 +1138,16 @@ paths:
       operationId: PostMastersMasterNumberConfiguration
       tags:
         - masters
-      summary: Write the configuration of the specified Master.
-      description: Write the configuration of the specified Master.
+      summary: Write the configuration of the specified W-Master.
+      description: Write the configuration of the specified W-Master.
```

Comment:
I disagree on this change, since it applies also for wired masters

Reply:
For now or with the version I used IO-Link Wired has no Master configuration. Regarding of this I wrote W-Master to be clear that this configuration is only for Wireless.

Reply:
Please create also a configuration example for the wired case (an empty data object will do for the beginning).